### ISSUE TITLE
Bisect bug comment should include units

### ISSUE THREAD

**Issue by anniesullie** [catapult-project/catapult#3667]
When the bisect makes a bug comment, it puts absolute values in with no units. Is it possible to add units? It would have helped the user in https://bugs.chromium.org/p/chromium/issues/detail?id=734619#c9 for example.

@simonhatch

**Comment by simonhatch** [comment]
Yeah the original chartjson has units in it, I can probably pull that and display in the output.
